net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@(PACKAGE_wireless-tools||PACKAGE_iw) +ethtool +procps +CONFIG_USB_SUPPORT:usbutils +CONFIG_PCI_SUPPORT:pciutils
```

Comment:
Hi,

The current status is that I have opened a PR to aircrack-ng (https://github.com/aircrack-ng/aircrack-ng/pull/57) and that the PR somehow failed OSX/Clang and some MIPS routers...

sorry for being silent for so long.  things are piling up here.